pasture-io/Cargo.toml

```diff
@@ -16,6 +16,7 @@ readme = "README.md"
 [dependencies]
 pasture-core = {version = "=0.1.0", path = "../pasture-core" }
 pasture-derive = {version = "=0.1.0", path = "../pasture-derive"}
+nalgebra = {version = "0.23.1", features = ["serde-serialize"] }
```

Comment:
`pasture-core` re-exports `nalgebra`, so why are you adding it here explicitly as dependency?

Reply:
That's for sure is an unnecessary import. Its removed.